docs/docusaurus/docs/core/installation_and_setup/components_how_to_configure_a_validation_result_store_in_amazon_s3/_preface.mdx

```diff
@@ -0,0 +1,16 @@
+import Prerequisites from '../../../connecting_to_your_data/components/prerequisites.jsx'
```

Comment:
I would go through and delete all of these `import Prerequisites from` statements, and then delete the corresponding `<Prerequisites>` and `</Prerequisites>` tags -- update the prerequisites sections to just use a H2 `## Prerequisites` and a list of items.  Remove the existing links from those items because they point at OSS docs that don't exist.  Update them with the new links to the /core/ docs, instead.  That should also resolve a lot of the broken link errors.

Reply:
@Rachel-Reverie - done